maintainers/maintainer-list.nix

```diff
@@ -16638,6 +16638,14 @@
     githubId = 8577941;
     name = "Kevin Rauscher";
   };
+  tomasajt = {
```

Comment:
No e-mail?

Reply:
I don't currently have a business email address, when I do, I'll add it